Car_owners/car_owners/prod_settings.py

```diff
@@ -121,3 +117,12 @@
 SESSION_COOKIE_SECURE = False
 CSRF_COOKIE_SECURE = False
 SECURE_SSL_REDIRECT = False
+
+# Manage static files while DEBUG=False
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
+
+# Override production variables if DJANGO_DEVELOPMENT env variable is True
```

Comment:
https://docs.djangoproject.com/en/4.1/topics/settings/#designating-the-settings just use this